lib/scarpe/wv/line.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require_relative "shape_helper"
+
+class Scarpe
+  class WebviewLine < Scarpe::WebviewWidget
+    include ShapeHelper
+
+    def initialize(properties)
+      super(properties)
+    end
+
+    def element
+      HTML.render do |h|
+        h.div(id: html_id, style: style) do
+          h.svg(width: @x2, height: @y2) do
+            h.line(x1: @left, y1: @top, x2: @x2, y2: @y2, style: line_style)
+          end
+        end
+      end
+    end
+
+    private
+
+    def style
+      {
+        left: "#{@left}px",
+        top: "#{@top}px",
+      }
+    end
+
+    def line_style
+      {
+        stroke: "red",
```

Comment:
Note: stroke is static here as it probably need us to figure out how we want shapes to structure (same as fill) 

for both fill and stroke i assume we need to restructure how we write shapes in shoes

Reply:
The examples I see with (e.g.) arc and things use an outer "shape do" block. That would give more structure -- I'm worried these will stack funny since each is in its own individual svg block. But I figure if I care enough, I can go in and fix that... I haven't made it a personal project yet :-)